### ISSUE TITLE
Post-login destination

### ISSUE THREAD

**Issue by jennydaman** [FNNDSC/ChRIS_store_ui#132]
Same issue as described here
https://github.com/FNNDSC/ChRIS_ui/issues/225

**Comment by DivyanshiSingh** [comment]
Can I work on this? @jennydaman

**Comment by jennydaman** [comment]
Would you like to work on this at the same time as Patternfly4 buttons, or one or the other?

Whoever takes up this task should bounce ideas with @archanaserver on https://github.com/FNNDSC/ChRIS_ui/issues/225

**Comment by DivyanshiSingh** [comment]
Yes, I can take this up in parallel as I have updated the react-router to v5 it's more easy to implement, either we can include it in pf4 changes or I can raise a separate MR. Any suggestions @jennydaman ? 
It's just a small change for pf4, if you agree I can straight away raise the PR and you can review it.

**Comment by jennydaman** [comment]
New feature on the pf4 branch sounds better.
